app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with model: @task do |f| %>
+    <p>Please provide the following information to edit your book</p>
+
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+
+    <%= f.label :description %>
+    <%= f.text_field :description %>
+
+    <%= f.label :completed_at %>
+    <%= f.text_field :completed_at %>
```

Comment:
Consider whether this should be entered by the user. Instead, it would make sense to set this as `nil` initially, and change to `Time.now` when it's marked completed (as you do).